### PR TITLE
update vip invitations for dc30

### PR DESCRIPTION
**Pull request by BenGardiner**

### CODE REVIEW

File: texts/guest_invitation.inc [hunk 1]
```diff
@@ -6,7 +6,9 @@ We love the work you do and would be grateful to have you bring something to sha
 
 If you are interested, our CFP is live now: $CFP_link
 
-For DEF CON 29, things are still in a bit of a state of flux. However, we are moving forward on the assumption that the DEF CON hybrid event will take place. Similar to Safe Mode DEF CON 28, all talks and events are going to be pre-recorded to be able to stream online and play in-person. Keep an eye on the DEF CON website for updates!
+Also, this year we will be running a "Make Your Own Use" contest! We are looking for judges, would you be a judge? The concept of the contest is submissions can be made based on any project that worked on over the year, there may be 3 categories for a) repairing a closed device, b) adding a feature to a device that was never intended, c) bypassing DRM on devices to unlock their full potential -- if there are many submissions. We expect the judges to award points based on 1) how bent/changed/diferent/unintended it is 2) re-work quality 3) re-work ingenuity 4) weirdness (it is DEF CON).
```
Comment (kbembedded) [inline texts/guest_invitation.inc]: Suggest changing "would you be a judge" to "would you be interested in volunteering to be a judge"

**Comment by kbembedded** [comment]
Aside from comment above, looks good to me.

File: texts/guest_invitation.inc (hunk 1 above)
Comment (BenGardiner) [reply texts/guest_invitation.inc]: good note, thanks bunni

**Review by ShortTie — CHANGES_REQUESTED** [review]
There are a number of events in the planning stages. Rather than specify judges for a specific event, I would suggest that we say we are looking for judges for some contests, and then we can assign them to what they match up. Something like

"This year a number of contests need judges. Rather than set up an arbitrary panel, we want to encourage any that want to participate to sign up to be a judge. We will have specific details of events coming out over the next few months.  As we assign judges to specific events we will work with you on expectations and time requirements."

**Review by ShortTie — CHANGES_REQUESTED** [review]
Sorry, I didn't check my spelling on my suggestion "Rugoldberg" should be "Rube Goldberg". Any other spelling issues I am missing? Otherwise I think it is good to go.

**Comment by kbembedded** [comment]
> Any other spelling issues I am missing? Otherwise I think it is good to go.

Re-read, looks good to me.

**Pull request merged**